### DISCUSSION TITLE
Lightning addresses on any domain

### DISCUSSION THREAD

**Discussion by iBobik** [btcpayserver/btcpayserver#3184] [Ideas / Feature Requests]
Currently it is possible to configure [internet identifiers (email-like addresses)](https://github.com/fiatjaf/lnurl-rfc/blob/luds/16.md) only on main domain of BTCPay server. It event does not support additional domains (BTCPAY_ADDITIONAL_HOSTS).

Ideally I would like to configure it for any domain, even domains not connected to BTCPay server (e.g. with some website):
1. Integrations → Lightning Address Setup → Add Address
2. Select External domain
3. Write whole address e.g. donation@example.com
4. Message like: "BTCPay server is accepting donation@example.com now, but you need to setup redirect to https://btcpay.example.com/external-ln-address/donation@example.com from https://example.com/.well-known/lnurlp/donation"

It will also be nice to customize metadata field:
- checkbox if address is also e-mail address (metadata will not use text/identifier but text/email)
- upload image (image/png or image/jpeg)
- custom text/plain and text/long-desc

**Comment by andysavage** [comment]
Is this possible yet? It's a few years old so maybe....

**Reply by dennisreimann** [reply]
See this FAQ entry: [How to redirect a Lightning Address?](https://docs.btcpayserver.org/FAQ/LightningNetwork/#how-to-redirect-a-lightning-address)

**Reply by andysavage** [reply]
> See this FAQ entry: [How to redirect a Lightning Address?](https://docs.btcpayserver.org/FAQ/LightningNetwork/#how-to-redirect-a-lightning-address)

Great thanks. Will that work with a different domain as well? e.g btcpayserver.domain1.com having lightning addresses at @domain2.com ?

No worries if not, just would be handy.

**Reply by dennisreimann** [reply]
Yes, you can use sevral domains and addresses, redirecting to a single canonical LN Address on your BTCPay Server instance.

**Reply by andysavage** [reply]
Thanks for your time!
